core/src/main/java/com/linecorp/armeria/common/StringMultimap.java

```diff
@@ -629,17 +629,24 @@ final void add(IN_NAME name, String value) {
         requireNonNull(value, "value");
         final int h = hashName(normalizedName);
         final int i = index(h);
-        add0(h, i, normalizedName, value);
+        add0(h, i, normalizedName, value, true);
     }
 
     final void add(IN_NAME name, Iterable<String> values) {
+        add(name, values, true);
+    }
+
+    final void add(IN_NAME name, Iterable<String> values, boolean notifyChange) {
```

Comment:
How about making this method private and adding two variants:
- `add()` and `set()` with old signature
- `addAndNotify()` and `setAndNotify()`

Reply:
I think the default behavior of `add()` and `set()` are `add` or `set` and notify.
If we add `{add,set}AndNotify`, we have to change all setters.
How about adding `addOnly()` and `setOnly()`?

Reply:
Ah, OK. `add/setWithoutNotifying()`?